package.json

```diff
@@ -120,5 +122,11 @@
       "babel-runtime",
       "jscodeshift"
     ]
-  }
+  },
+  "bundlesize": [
+    {
+      "path": "./dist/preact.min.js",
+      "threshold": "4Kb"
```

Comment:
I'm pretty sure it's the maximum size allowed for the lib. I believe it will just diff against the last-known size of `master` branch & do diff on `current - master`